backend/internal/config.go

```diff
@@ -145,3 +168,46 @@ func LoadConfig() (Configuration, error) {
 
 	return config, nil
 }
+
+func expandPath(path string) (string, error) {
```

Comment:
The function `expandPath` is doing multiple things, what do u think of splitting it ?